packages/api/tests/endpoints/workflows.js

```diff
@@ -122,20 +120,3 @@ test('GET with path parameters returns a 404 for a nonexistent workflow', async
   t.is(response.status, 404);
   t.is(response.body.message, 'Workflow does not exist!');
 });
-
-test.serial('GET /good-workflow returns a 404 if the workflows list cannot be fetched from S3', async (t) => {
```

Comment:
Why was this test removed?

Reply:
Now that the list of valid users is stored in S3 (in the `system_bucket`), this case could not come up. If the system bucket does not exist, you are going to get a 400 response back because it is unable to authenticate you before it even makes it this far. If the bucket _does_ exist, then either 1) the workflow exists, in which case you'll get it back, or 2) the work flow does not exist, which is tested in the previous test.